dependency/controller.py

```diff
@@ -24,22 +25,20 @@ def responsify_dependency(dependency):
 
 @blueprint.post('/')
 def create_dependency(workspace_id: str):
-    dependee_id = request.json['dependee_id']
-    dependent_id = request.json['dependent_id']
-    manner = request.json['manner']
-
+    try:
+        dependee_id = str(request.json['dependee_id'])
```

Comment:
I'm not sure if we should convert to string here since this will never fail and could easily mask issues that would otherwise be raised in the data layer (say for example they pass a list, str will happily turn it into one rather than raise an error about that being undefined behavior)